core/tests/protractor/learnerFlow.js

```diff
@@ -54,12 +54,15 @@ describe('Learner dashboard functionality', function() {
     await waitFor.pageToFullyLoad();
   };
 
-  var createDummyExplorationOnDesktopAsAdmin = async function() {
+  var createDummyExplorationOnDesktopAsAdmin = async function(
+      expectWelcomeModal) {
```

Comment:
Here and everywhere else in this PR, avoid starting a variable name with a verb, we use verbs for function names (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript)).

Maybe this can be names `welcomeModalIsShown`?

Reply:
Done!